packages/vite/src/node/server/index.ts

```diff
@@ -920,7 +920,7 @@ export async function _createServer(
   }
 
   // error handler
-  middlewares.use(errorMiddleware(server, !!middlewareMode))
+  middlewares.onError = errorMiddleware(server, !!middlewareMode)
```

Comment:
This is also a big breaking change I think. It will limit users to only able to use Polka? If they prefer to use Express, could they still do so?

Reply:
do you have an example?

you mean they'd create a vite instance and overwrite the middleware with express?

Reply:
Let's say if we use the previous express setups in the `playground/` and keep the polka changes from core, do they still work together? My fear is that if the have:

```js
const app = express()
app.use(vite.middlewares)
```

The error handling would no longer work. Or maybe I'm mistaken.